tests/Type/data/model-factories.php

```diff
@@ -7,81 +7,70 @@
 
 use function PHPStan\Testing\assertType;
 
-function doFoo(): void
```

Comment:
`assertType` calls should always be in a dummy function. Please revert.

Reply:
@canvural, why is that? We have several test files in this project that don't follow that convention:

- tests/Type/data/environment-helper.php
- tests/Type/data/translator.php
- tests/Type/data/request-user.php
- tests/Type/data/collection-stubs.php
- ...

**All** of the type test files in the Laravel framework that don't follow that:
- https://github.com/laravel/framework/blob/48246da2320c95a17bfae922d36264105a917906/types/Support/Helpers.php#L5

And the same thing with PHPStan tests:
- https://github.com/phpstan/phpstan-src/blob/f9361865944f2da686049fcdeef173065052d82b/tests/PHPStan/Analyser/data/round-php8.php#L13


I can revert if you want, but I feel that the dummy function is unnecessary, adds an extra indentation level, and the function name doesn't even describe the tests being performed.

Reply:
I'd rather get clarification from Ondre before we just call his own tests wrong...that performance issue could have been fixed

Reply:
The example file you gave is 2 years old. It's normal it's not updated.

Reply:
I wrapped the tests in a closure